spec/models/issue_spec.rb

```diff
@@ -18,43 +18,50 @@
 require 'spec_helper'
 
 describe 'User' do
-	describe 'Permissions' do
-		let!(:user) { create :user }
-		let!(:admin) { create :user, :admin }
-		let(:issue) { Issue.new }
-
-		describe 'can_show?' do
-			it 'returns true for author' do
-				issue.author = user
-				expect(issue.can_show? user).to be_true
-			end
-			it 'returns true for admin' do
-				expect(issue.can_show? admin).to be_true
-			end
-			it 'returns false if neither admin nor author' do
-				expect(issue.can_show? user).to be_false
-			end
-		end
-		describe 'can_create?' do
-			it "returns true" do
-				expect(issue.can_create? nil).to be_true
-			end
-		end
-		describe 'can_update?' do
-			it 'returns true for admin' do
-				expect(issue.can_update? admin).to be_true
-			end
-			it 'returns false for non-admin' do
-				expect(issue.can_update? user).to be_false
-			end
-		end
-		describe 'can_destroy?' do
-			it 'returns true for admin' do
-				expect(issue.can_destroy? admin).to be_true
-			end
-			it 'returns false for non-admin' do
-				expect(issue.can_destroy? user).to be_false
-			end
-		end
-	end
-end 
\ No newline at end of file
+  describe 'Permissions' do
+    let!(:user) { create :user }
+    let!(:admin) { create :user, :admin }
+    let(:issue) { Issue.new }
+
+    describe 'can_show?' do
+      it 'returns true for author' do
+        issue.author = user
+        expect(issue.can_show? user).to be_truthy
+      end
+
+      it 'returns true for admin' do
+        expect(issue.can_show? admin).to be_truthy
+      end
+
+      it 'returns false if neither admin nor author' do
+        expect(issue.can_show? user).to be_falsey
+      end
+    end
+
+    describe 'can_create?' do
+      it "returns true" do
+        expect(issue.can_create? nil).to be_truthy
+      end
+    end
+
+    describe 'can_update?' do
+      it 'returns true for admin' do
+        expect(issue.can_update? admin).to be_truthy
+      end
+
+      it 'returns false for non-admin' do
+        expect(issue.can_update? user).to be_falsey
+      end
+    end
+
+    describe 'can_destroy?' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.